clients/ui/frontend/src/app/pages/modelRegistry/screens/RegisterModel/RegisterModelDetailsFormSection.tsx

```diff
@@ -17,12 +16,14 @@ type RegisterModelDetailsFormSectionProp<D extends RegisterModelFormData> = {
   setData: UpdateObjectAtPropAndValue<D>;
   hasModelNameError: boolean;
   isModelNameDuplicate?: boolean;
+  isCatalogModel?: boolean;
 };
 const RegisterModelDetailsFormSection = <D extends RegisterModelFormData>({
   formData,
   setData,
   hasModelNameError,
   isModelNameDuplicate,
+  isCatalogModel = false,
```

Comment:
There are a few other things this `isCatalogModel` prop should do here - compare with the original model catalog in ODH. If you're registering a catalog model, the model location section should disable the radio buttons to force you to choose "URI", and it should change the URI input field to be static text. The user can't change the model location that's coming from the catalog.